progress2.inc

```diff
@@ -109,6 +108,7 @@ stock DestroyPlayerProgressBar(playerid, PlayerBar:barid) {
 	return 1;
 }
 
+stock bool:IsPlayerProgressBarVisible(playerid, PlayerBar:barid) return (IsValidPlayerProgressBar(playerid, barid) && pbar_Data[playerid][_:barid][pbar_show]);
```

Comment:
Please use braces to match the existing style.

Reply:
i don't understand what is the problem with the braces...

Reply:
Because when the function is all on a single line, it makes it difficult to read.

For example, I can't quickly see what part of the line is the function signature and what part is the actual body of the function and return value. Typically, it's best to always use braces for statements regardless of how many lines there are.

It's also good practice to be consistent in a codebase so your eyes must work less to parse the code. This also matters when contributing to other codebases!

Thanks!

Reply:
Okay iam sorry for that... going to edit it...